spec/jekyll_seo_tag_integration_spec.rb

```diff
@@ -23,6 +23,11 @@
     expect(output).to match(%r!Jekyll SEO tag v#{version}!i)
   end
 
+  it "outputs jekyll version" do
+    jekyll_version = Jekyll::VERSION
+    expect(output).to match(%r!Jekyll v#{jekyll_version}!i)
```

Comment:
You can use `#{Jekyll::VERSION}` here. You don't use `jekyll_version` anywhere else so there's no need for the variable.